build.gradle

```diff
@@ -69,6 +74,11 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.10.1'
     testImplementation 'org.junit.jupiter:junit-jupiter-params:5.10.1'
     testImplementation 'org.junit.platform:junit-platform-engine:1.10.1'
+    modules {
+        module("com.google.collections:google-collections") {
+            replacedBy("com.google.guava:guava", "Google Collections is part of Guava")
+        }
+    }
```

Comment:
Please remove this module section and don't upgrade to checkstyle v10.12.6. The issue with the transitive dependency is on their side and will hopefully be fixed with v10.12.7. I reverted the change done by the renovate bot.

Reply:
fixed